src/tasks/package.js

```diff
@@ -32,20 +33,21 @@ module.exports = () => {
         node: '>=4.3.0 <5.0.0 || >=5.10',
       },
       scripts: {
-        prepublish: 'yarn run build',
-        prebuild: 'yarn run clean:dist',
+        start: 'yarn run serve:dev src',
         build: "cross-env NODE_ENV=production babel -s true src -d dist --ignore 'src/**/*.test.js'",
         'clean:dist': 'del-cli dist',
-        start: 'yarn run serve:dev src',
-        'serve:dev': 'nodemon $2 --exec babel-node',
         lint: 'eslint --cache src test',
+        'lint-staged': 'lint-staged',
+        prebuild: 'yarn run clean:dist',
+        prepublish: 'yarn run build',
+        release: 'yarn run standard-version',
         security: 'nsp check',
+        'serve:dev': 'nodemon $2 --exec babel-node',
         test: 'jest',
         'test:coverage': "jest --collectCoverageFrom='src/**/*.js' --coverage",
-        'travis:test': 'yarn run test',
         'travis:coverage': 'yarn run test:coverage',
         'travis:lint': 'yarn run lint && yarn run nsp',
-        'lint-staged': 'lint-staged',
+        'travis:test': 'yarn run test',
```

Comment:
We test on 3 versions of node, we only want to collect coverage once per commit hash which is why those two tasks are split.